packages/fxa-content-server/app/scripts/views/app.js

```diff
@@ -181,10 +186,30 @@ var AppView = BaseView.extend({
         childView.model.set(options.model.toJSON());
         childView.displayStatusMessages();
 
+        this._showSurvey(this._currentView, options);
+
         return childView;
       });
   },
 
+  /**
+   * Get a survey from the survey targeting module.  If there's a survey,
+   * append it to the view.
+   *
+   * @param {Backbone.View} view the current (parent) view
+   * @param {Object} options the option for the view to be displayed
+   */
+  _showSurvey(view, options) {
+    if (this._surveyTargeter) {
+      this._surveyTargeter.getSurvey(options.viewName).then(survey => {
+        if (survey) {
+          survey.render();
+          view.$el.append(survey.el);
+        }
+      });
+    }
+  },
```

Comment:
errr did we just straight up forgot to write tests for this?  We should add tests for it.

Reply:
What do you think we should be testing here? That surveyTargeter exists or that a survey exists before we call render? We have significant coverage around surveyTargeter file and surveyView

Reply:
We could test to see

- if _showSurvey is called
- behavior when all the options should return a survey but _surveyTargeter is undefined, 
- behavior when all the options should return a survey (happy path)

Reply:
Created FXA-2003 to address this.